src/includes/modules/payment/payment_rth_stripe.php

```diff
@@ -17,13 +17,12 @@
 declare(strict_types=1);
 
 use RobinTheHood\ModifiedStdModule\Classes\StdModule;
-use RobinTheHood\Stripe\Classes\Order;
-use RobinTheHood\Stripe\Classes\Session;
+use RobinTheHood\Stripe\Classes\{Order, Session, Constants};
 
 class payment_rth_stripe extends StdModule
 {
     public const VERSION = '0.1.0';
-    public const NAME    = 'MODULE_PAYMENT_PAYMENT_RTH_STRIPE';
+    public const NAME    = Constants::MODULE_NAME;
```

Comment:
Unfortunately this is not possible. Since system modules and payment modules require different names.
Maybe split into: Constants::MODULE_NAME_SYSTEM and Constants::MODULE_NAME_PAYMENT